katalog/elasticsearch-triple/triple.yml

```diff
@@ -9,5 +13,5 @@ spec:
       containers:
         - name: elasticsearch
           env:
-            - name: discovery.zen.ping.unicast.hosts
-              value: "elasticsearch-0.elasticsearch,elasticsearch-1.elasticsearch,elasticsearch-2.elasticsearch"
+            - name: cluster.initial_master_nodes
+              value: "elasticsearch-0,elasticsearch-1,elasticsearch-2"
```

Comment:
Does it make sense to use the DNS based discovery? This way, we could leverage the headless Kubernetes service populated by the StatefulSet.

https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-discovery-hosts-providers.html#built-in-hosts-providers

Reply:
🤔 Don't know what's the difference with the current behavior

Reply:
I may be wrong but I think that you cannot leverage the headless service when you are using `cluster.initial_master_nodes` (which is the first time you start the cluster, ever, clean volumes I believe).

You can use the headless service with `discovery.seed_hosts` but maybe we could call it `elasticsearch-headless` instead of just `elasticsearch` ^^

Reply:
Currently, the `elasticsearch` service is of the type `headless`:

```yaml
clusterIP: None
```

This didn't change in this release. We are not deploying a non-headless service.